.firebaserc

```diff
@@ -1,5 +1,6 @@
 {
   "projects": {
-    "default": "act-now-links-dev"
+    "develop": "act-now-links-dev",
+    "production": "act-now-links-dev"
```

Comment:
We will want to set up an `act-now-links-prod` firebase project, but until then just point the production project at dev.